lumicks/pylake/force_calibration/calibration_item.py

```diff
@@ -3,13 +3,43 @@
 from functools import wraps
 from collections import UserDict
 
+from lumicks.pylake.channel import empty_slice
+from lumicks.pylake.force_calibration.convenience import calibrate_force
 from lumicks.pylake.force_calibration.calibration_models import DiodeCalibrationModel
 from lumicks.pylake.force_calibration.detail.calibration_properties import (
     CalibrationPropertiesMixin,
 )
 
 
 class ForceCalibrationItem(UserDict, CalibrationPropertiesMixin):
+    def __init__(
+        self,
+        dictionary=None,
+        *,
+        voltage=None,
+        sum_voltage=None,
+        driving=None,
+    ):
+        super().__init__(dictionary)
+        self._voltage = voltage
+        self._sum_voltage = sum_voltage
+        self._driving = driving
+
+    @property
+    def voltage(self):
+        """Uncalibrated voltage reading on the detector"""
+        return self._voltage if self._voltage else empty_slice
```

Comment:
generally  for things like this I test explicitly `if x is not None`. I don't expect it to happen but if somehow `self._voltage` becomes something falsey it might be harder to track down because you'll get an empty slice which might look ok. Again, I don't know if this'll be a problem, just my personal preference

Reply:
Yeah, I'm aware of the distinction. In this particular case, I intended falsey. We would get either `None` or a Slice with no data (an empty slice).

I'll make it a little more explicit and check upstream for the emptiness and then only check `None` case here. I agree that that is probably a little safer.